lib/jekyll/drops/drop.rb

```diff
@@ -211,6 +212,12 @@ def fetch(key, default = nil, &block)
         return yield(key) unless block.nil?
         return default unless default.nil?
       end
+
+      private
+
+      def fallback_data
+        @fallback_data ||= {}
+      end
```

Comment:
The base drop class (in `#[]`, `#keys` and `#key?`) assumes fallback data to exist and return a hash. Stubbed out the default value so that child classes don't each have to implement it if they don't have fallback data (including our test fixture).

Reply:
I intentionally left this method out so all drops _had_ to implement this – it was part of the design of Jekyll Drops, that some fallback data had to be provided by the developer of a drop. I'm not sure I like that going away. :/

Reply:
> it was part of the design of Jekyll Drops, that some fallback data had to be provided by the developer of a drop

Admittedly out of scope, so glad to remove from this PR. Curious if that rationale still holds true when Jekyll internally often implements it as an empty hash?

Reply:
Removed via 847ed7f